src/numerics/BandMatrix.cpp

```diff
@@ -10,19 +10,11 @@
 #if CT_USE_LAPACK
     #include "cantera/numerics/ctlapack.h"
 #else
-    #if CT_SUNDIALS_USE_LAPACK
-        #if CT_SUNDIALS_VERSION >= 30
-            #include "sunlinsol/sunlinsol_lapackband.h"
-        #else
-            #include "cvodes/cvodes_lapack.h"
-        #endif
+    #if CT_SUNDIALS_VERSION >= 30
```

Comment:
the logic appears to be different here. The PR comment states that it was simplified ...

Reply:
Hmmm good point. It's simplified in the sense that if `CT_USE_LAPACK` is false, then `CT_SUNDIALS_USE_LAPACK` is also always false, which means the first branch in the `# else` clause could never be followed. I'll clarify the commit message.

Reply:
Beyond, please feel free to ignore my 'Accept' - per discussion elsewhere, this is probably more a comment.